src/main/modules/Execution.js

```diff
@@ -0,0 +1,41 @@
+var index2 = 0;
+export class Execution {
+    constructor(parserActionsArray, parserActorsArray){
+        this.parserActionsArray = parserActionsArray;
+        this.parserActorsArray = parserActorsArray;
+    }
+
+    execute(parserActionsArray, parserActorsArray){
+        for (var index = 0; index < (parserActionsArray.length); index++) {
+            switch(parserActionsArray[index].actionType){
```

Comment:
For this iteration I think it works but I also think that is better if you create an external class, for example "DefineAction" to have more abstration and accomplish the IO SOLID principle

An example of this class can be:

`class DefineAction {
  constructor(actionType = parserActionsArray[index].actionType){
    this.actionType = actionType;
  }

  do () {  }
}

export class DefinePickWeaponAction extends DefineAction {
  do () {
    if (parserActorsArray[index2].name == parserActionsArray[index].actionActor){
      parserActorsArray[index2].weapon = parserActionsArray[index].element;
    }
  }
}
`

Reply:
Understood Daniel, thank you for the feedback, we will try to implement this structure for next iterations.